core/field_textinput.js

```diff
@@ -531,8 +531,11 @@ Blockly.FieldTextInput.prototype.resizeEditor_ = function() {
  * @deprecated
  */
 Blockly.FieldTextInput.numberValidator = function(text) {
-  console.warn('Blockly.FieldTextInput.numberValidator is deprecated. ' +
-               'Use Blockly.FieldNumber instead.');
+  Blockly.utils.deprecation.warn(
+    'FieldTextInput.numberValidator',
+    'May 2019',
+    'October 2020',
+    'Blockly.FieldNumber');
```

Comment:
Thanks!

Just a few comments then should be good : ) 

I think eslint is unhappy because these should be indented two more spaces. Just as a heads up you can run eslint locally by using `npm run lint`.

Since the second date is the deletion date, can we make it December 2020? I think there are quite a few other field methods that are scheduled for deletion in December so we can go back and delete in same PR.

Reply:
Done!

Reply:
Now it looks good 😄